src/features/colinks/EthDenverContestBanner.tsx

```diff
@@ -0,0 +1,47 @@
+import { client } from 'lib/gql/client';
+import { useQuery } from 'react-query';
+import { NavLink } from 'react-router-dom';
+
+import HintBanner from 'components/HintBanner';
+import { coLinksPaths } from 'routes/paths';
+import { Flex, Text, Button } from 'ui';
+
+export const EthDenverContestBanner = ({ address }: { address: string }) => {
+  const { data: checkEthDenverInvitee } = useQuery(
+    ['colink_checkEthDenverInvitee', address, 'wizard'],
```

Comment:
@wingfeatherwave general best practice is to have all the constants be first in the query key, that allows for more modular query key invalidation.